prime-router/src/test/kotlin/fhirengine/engine/FhirTranslatorTests.kt

```diff
@@ -68,7 +70,7 @@ class FhirTranslatorTests {
                 Topic.FULL_ELR,
                 CustomerStatus.ACTIVE,
                 ORU_R01_SCHEMA,
-                format = Report.Format.HL7_BATCH,
+                format = Report.Format.HL7,
```

Comment:
Why did this need to change?

Reply:
Will revert these changes, I missed these during cleanup. Good catch!